doc/sphinx/addendum/type-classes.rst

```diff
@@ -295,33 +295,37 @@ the Existing Instance command to achieve the same effect.
 Summary of the commands
 -----------------------
 
-.. cmd:: Class @inductive_definition {* with @inductive_definition }
+.. cmd:: Class @record_definition
+         Class @class_definition
```

Comment:
```suggestion
         Class @singleton_class_definition
```

Reply:
I've committed the fix to make the build of the doc pass.

Reply:
An example of a command for which doc_grammar can't update the syntax automatically because there are two matches for the command name: `Class @record_definition` and `Class @singleton_class_definition`.  And not doing N+1 builds :-(

Reply:
> An example of a command for which doc_grammar can't update the syntax automatically because there are two matches for the command name: `Class @record_definition` and `Class @singleton_class_definition`.

Too bad, but then maybe we should transform this into `Class {| @record_definition | @singleton_class_definition }`. WDYT?

Reply:
I already print a warning if the rst doesn't exactly match a command/tactic (ignoring subscripts).  Probably time to make that an error, which I'll do now.  We could tweak this single case, but there are other cases where the right hand sides are too long to combine.